routing/world_graph.hpp

```diff
@@ -19,8 +20,9 @@ class WorldGraph final
   enum class Mode
   {
     SingleMwm,
-    WorldWithLeaps,
-    WorldWithoutLeaps,
+    LeapsOnly,
```

Comment:
Please, add a comment what does LeapsWithPossible mean, and what's the difference with LeapsOnly and NoLeaps.